modules/python/python-logparser.c

```diff
@@ -182,10 +182,10 @@ python_parser_process(LogParser *s, LogMessage **pmsg, const LogPathOptions *pat
   {
     LogMessage *msg = log_msg_make_writable(pmsg, path_options);
 
-    msg_debug("Invoking the Python parse() method",
+    msg_debug("python-parser message processing started",
+              evt_tag_str ("input", input),
               evt_tag_str("parser", self->super.name),
               evt_tag_str("class", self->class),
-              log_pipe_location_tag(&self->super.super),
```

Comment:
Why is the location tag deleted here?

Reply:
In my previous comment I have described why it is deleted from here.